konduit-serving-pipeline/src/test/java/ai/konduit/serving/pipeline/impl/data/DataTest.java

```diff
@@ -112,12 +125,117 @@ public void testEmbeddedData() {
         Data layeredContainer = JData.singleton("upperLevel", ndContainer);
     }
 
-    @Ignore
     @Test
     public void testSerde() {
         Data someData = JData.singleton(KEY, Long.valueOf(200));
-        someData.save(new File("temp"));
-        Data restoredData = JData.fromFile(new File("temp"));
+        try {
+            someData.save(new File("temp"));
+        } catch (IOException e) {
+            log.error("IOException in DataTest.testSerde()", e);
+        }
+        Data restoredData = null;
+        try {
+            restoredData = JData.fromFile(new File("temp"));
+        } catch (IOException e) {
+            log.error("IOException in DataTest.testSerde()", e);
+        }
+        assertEquals(someData.get(KEY), restoredData.get(KEY));
+    }
+
+    @Test
+    public void testConvertToBytes() {
+        JData someData = (JData) JData.singleton(KEY, Long.valueOf(200));
+        byte[] output = someData.asBytes();
+        assert(output != null);
+    }
+
+    @Test
+    public void testConvertToJson() {
+        Data someData = (JData) JData.singleton(KEY, Long.valueOf(200));
+        String jsonStr = someData.toJson();
+        String expected = "{\n" +
+                "  \"mapItems\": {\n" +
+                "    \"stringData\": {\n" +
+                "      \"iValue\": \"200\",\n" +
+                "      \"type\": \"INT64\"\n" +
+                "    }\n" +
+                "  }\n" +
+                "}";
+
+        assertEquals(expected, jsonStr);
+    }
```

Comment:
Test can go, JSON is implemented/tested elsewhere

Reply:
Removed test.